src/amo/components/SearchSort/SearchSort.scss

```diff
@@ -65,6 +73,10 @@
     background: transparentize($link-color, 0.8);
     box-shadow: none;
   }
+
+  @include respond-to(large) {
```

Comment:
Yeah, I thought that too... with sagas we can have the notion of server-versus-client-side sagas. One way to do this might be to have the UA and viewport be stored in state and only fire off actions/saga requests for certain bits of data if the viewport is big enough.

I think that's only useful if the perf gain outweighs all the complexities that go with such a solution, but I'd be okay with that.